solr/core/src/test/org/apache/solr/response/TestRetrieveFieldsOptimizer.java

```diff
@@ -411,21 +411,20 @@ static enum TEST_TYPE {
   }
 
   static final Map<TEST_TYPE, String> solrClassMap =
-      Collections.unmodifiableMap(
-          Stream.of(
-                  new SimpleEntry<>(TEST_TYPE.TINT, "solr.TrieIntField"),
-                  new SimpleEntry<>(TEST_TYPE.TLONG, "solr.TrieLongField"),
-                  new SimpleEntry<>(TEST_TYPE.TFLOAT, "solr.TrieFloatField"),
-                  new SimpleEntry<>(TEST_TYPE.TDOUBLE, "solr.TrieDoubleField"),
-                  new SimpleEntry<>(TEST_TYPE.TDATE, "solr.TrieDateField"),
-                  new SimpleEntry<>(TEST_TYPE.PINT, "solr.IntPointField"),
-                  new SimpleEntry<>(TEST_TYPE.PLONG, "solr.LongPointField"),
-                  new SimpleEntry<>(TEST_TYPE.PFLOAT, "solr.FloatPointField"),
-                  new SimpleEntry<>(TEST_TYPE.PDOUBLE, "solr.DoublePointField"),
-                  new SimpleEntry<>(TEST_TYPE.PDATE, "solr.DatePointField"),
-                  new SimpleEntry<>(TEST_TYPE.STRING, "solr.StrField"),
-                  new SimpleEntry<>(TEST_TYPE.BOOL, "solr.BoolField"))
-              .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+      Stream.of(
```

Comment:
No point in using Stream without any transformation.  Here, just call `Map.ofEntries`

Reply:
good catch - fixed in eb8e6ac281439f31ac3a59e247da5faccd7224dd